airflow/www/static/js/grid/details/content/taskInstance/Logs/index.jsx

```diff
@@ -0,0 +1,191 @@
+/*!
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React, { useRef, useState, useEffect } from 'react';
+import {
+  Text,
+  Box,
+  Flex,
+  Divider,
+  Code,
+  Button,
+  Checkbox,
+} from '@chakra-ui/react';
+
+import { getMetaValue } from '../../../../../utils';
+import LogLink from './LogLink';
+import useTaskLog from '../../../../api/useTaskLog';
+import LinkButton from '../../../../components/LinkButton';
+
+const showExternalLogRedirect = getMetaValue('show_external_log_redirect') === 'True';
+const externalLogName = getMetaValue('external_log_name');
+const logUrl = getMetaValue('log_url');
+
+const getLinkIndexes = (tryNumber) => {
+  const internalIndexes = [];
+  const externalIndexes = [];
+
+  [...Array(tryNumber + 1 || 0)].forEach((_, index) => {
+    if (index === 0 && tryNumber < 2) return;
+    const isExternal = index !== 0 && showExternalLogRedirect;
+    if (isExternal) {
+      externalIndexes.push(index);
+    } else {
+      internalIndexes.push(index);
+    }
+  });
+
+  return [internalIndexes, externalIndexes];
+};
+
+const Logs = ({
+  dagId,
+  dagRunId,
+  taskId,
+  executionDate,
+  tryNumber,
+  isGroup,
```

Comment:
We need to check if it is a group or if it is mapped. Mapped tasks still try to use the logs API:
<img width="564" alt="Screen Shot 2022-06-08 at 6 34 07 PM" src="https://user-images.githubusercontent.com/4600967/172728791-f73671b0-a048-4bde-9033-902955a5bd50.png">

Reply:
Good catch, I was  hidding the `Tab` but the `TabContent` was still mounted. I fixed just fixed that.

I also added a dag with mapped task in my test suite so I don't miss dynamic dag related issues.